plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -22,45 +22,62 @@ export function limitRenderer(group: ServiceGroup) {
   const limitNumber = roles - groupRoles;
 
   let icon = null;
-  if (limit === "Enforced") {
-    icon = (
-      <div className="table-content-spacing-right table-content-inline-block">
-        <Icon color={blue} shape={SystemIcons.CircleCheck} size={iconSizeXs} />
-      </div>
-    );
-  } else if (limit === "Partially Enforced") {
-    icon = (
-      <div className="table-content-spacing-right table-content-inline-block">
-        <Tooltip
-          id="quota-limit-tooltip"
-          trigger={
-            <Icon
-              color={greyLight}
-              shape={SystemIcons.CircleMinus}
-              size={iconSizeXs}
+  switch (limit) {
+    case "Enforced":
+      icon = (
+        <div className="table-content-spacing-right table-content-inline-block">
+          <Icon
+            color={blue}
+            shape={SystemIcons.CircleCheck}
+            size={iconSizeXs}
+          />
+        </div>
+      );
+      break;
+    case "Partially Enforced":
+      icon = (
+        <div className="table-content-spacing-right table-content-inline-block">
+          <Tooltip
+            id="quota-limit-tooltip"
+            trigger={
+              <Icon
+                color={greyLight}
+                shape={SystemIcons.CircleMinus}
+                size={iconSizeXs}
+              />
+            }
+          >
+            <Plural
```

Comment:
@TattdCodeMonkey , Plural is broken in Chinese, this is why I didn't use it.
Example (notice the "s" in English):
![Screenshot from 2019-07-15 09-07-40](https://user-images.githubusercontent.com/40791275/61197972-36930280-a6e0-11e9-8c68-6de39264b2b3.png)

Reply:
@GeorgiSTodorov 	where is that screenshot from? using a simple `Plural` i don't seem to be able to reproduce the `s` inbetween.

Reply:
@pierrebeitz , it's from the nodes table after you filter the nodes.

Reply:
On that instance we use a lingui-`Trans` and pluralize with our own implementation via `StringUtil.pluralize(name, totalLength)` which is hardcoded to english.

Lingui looks fine to me.

Reply:
Maybe I got confused and thought that it was broken there because of lingui.

Reply:
@pierrebeitz , another example of a broken translation in a place where we are using lingui Plural.
![Screenshot from 2019-07-15 12-35-22](https://user-images.githubusercontent.com/40791275/61207228-3b19e400-a6fd-11e9-8918-f624d418d499.png)

https://github.com/dcos/dcos-ui/blob/78f3b127ae4ce48a53f89f924346c55030e67b4e/plugins/services/src/js/columns/ServicesTableStatusColumn.tsx#L54

https://github.com/dcos/dcos-ui/blob/54a3114436bc127200165c956284d598fd4a7565/locale/zh/messages.json#L968